File: Instructor Java/Week07  Java/Week07 Java/Start.java
/**
 * Start
 */
public class Start {

  public static void main(String[] args) {

    var p = new Point(3, 4);
    System.out.println(p);

    var c = new Circle(3, 4, 5);
    System.out.println(c);
    c.setY(10);
    System.out.println(c);
    System.out.printf("Circle Area=%f\n", c.area());

    var sq = new Square(4, 14, 20);
    System.out.println(sq);
    sq.setX(10);
    System.out.println(sq);
    System.out.printf("Square Area=%f\n", sq.area());

    var rect = new Rect(4, 14, 20, 3);
    System.out.println(rect);
    rect.setS(10);
    System.out.println(rect);
    System.out.printf("Square Area=%f\n", rect.area());

  }
}

File: python/week07/Week07 Python/Start.py
from Point import Point
from Circle import Circle
from Square import Square


def start():
    p = Point(3, 4)
    print(p)
    p.x = 10
    print(p)
    print(f"Area= {p.area()}")

    c = Circle(3, 4, 5)
    print(c)
    c.r = 10
    print(c)
    print(f"Area= {c.area()}")

    sq = Square(3, 4, 5)
    print(sq)
    sq.y = 10
    print(sq)
    print(f"Area= {sq.area()}")


if __name__ == "__main__":
    start()
